src/locales/en.json

```diff
@@ -115,37 +115,37 @@
     },
     "audio-video": {
-      "title": "How does the Meta Search feature work for non-indexed media types?",
-      "demo-label": "Layout of Meta Search for non-indexed media type, like video",
+      "title": "How does the External Sources feature work for non-indexed media types?",
+      "demo-label": "Layout of External Sources for non-indexed media type, like video",
       "content": "{openverse} does not yet support built-in search for some media types, such as videos. For these media types, click on the appropriate tab and you’ll see sources where you can find CC-licensed content.",
-      "filter": "The Filters bar on the left includes the {use} filter, which may be supported by some meta search sources. Apply these before clicking on the button for the external source you’d like to search."
+      "filter": "The Filters bar on the left includes the {use} filter, which may be supported by some external sources sources. Apply these before clicking on the button for the external source you’d like to search."
     },
     "new": {
-      "title": "Can I suggest new sources for Meta Search?",
+      "title": "Can I suggest new sources for External Sources?",
       "content": "Yes, please! Create an {issue} in our GitHub repository or send us an {email} and tell us about the new sources you’d like to see included.",
       "issue": "issue",
       "email": "email"
     },
     "why": {
       "title": "Why did you build this?",
       "content": "For many years, CC has offered its users a dedicated search portal for searching platforms that have CC licensing filters built in. These platforms included Europeana, Google Images, Flickr, Jamendo, Open Clip Art Library, SpinXpress, Wikimedia Commons, YouTube, ccMixter, and SoundCloud. The search experience looked like this:",
-      "new": "For users of the legacy CC Meta Search site, the Meta Search feature on {openverse} will look familiar. The goal was to ensure that the functionality is not lost, but is updated and embedded within our new search engine for openly licensed content. In addition, the Meta Search feature builds on this functionality, allowing us to quickly add new sources to Meta Search as we discover them, and support new content types in the future, as we expand.",
+      "new": "For users of the legacy CC External Sources site, the External Sources feature on {openverse} will look familiar. The goal was to ensure that the functionality is not lost, but is updated and embedded within our new search engine for openly licensed content. In addition, the External Sources feature builds on this functionality, allowing us to quickly add new sources to External Sources as we discover them, and support new content types in the future, as we expand.",
```

Comment:
Should we leave the first as CC Meta Search? I guess that was the name when it was used, or was it simply CC Search (so, it would read 'users of the legacy CC Search site')

Reply:
Oh, and since we've removed the filters from external sources, it would probably not look familiar to the users of legacy CC Search site?